driver-core/src/main/java/com/datastax/driver/core/tracing/PrecisionLevel.java

```diff
@@ -0,0 +1,17 @@
+package com.datastax.driver.core.tracing;
+
+/** The precision level of tracing data that is to be collected. May be extended in the future. */
+public enum PrecisionLevel {
+  NORMAL(0),
+  FULL(1);
+
+  private final int precision;
+
+  PrecisionLevel(int precision) {
+    this.precision = precision;
+  }
+
+  public int comparePrecisions(PrecisionLevel precisionLevel) {
```

Comment:
Why not just order `NORMAL`, `FULL` in order of "precision" and just use the default `compareTo` enum method?